src/ios/CDVWKWebViewEngine.m

```diff
@@ -457,6 +457,19 @@ - (void) webView: (WKWebView *) webView decidePolicyForNavigationAction: (WKNavi
     return decisionHandler(NO);
 }
 
+#pragma mark - Plugin interface
+
+- (void)allowsBackForwardNavigationGestures:(CDVInvokedUrlCommand*)command;
+{
+    id value = [command.arguments objectAtIndex:0];
```

Comment:
I would use either of these commands: https://github.com/apache/cordova-ios/blob/077b1dc3c1d72ade9552773ed5a3a4c817800d37/CordovaLib/Classes/Public/CDVInvokedUrlCommand.h#L46-L50